components/Ui/UiCodeSnippet.vue

```diff
@@ -34,17 +34,12 @@ interface Props {
 
 const props = defineProps<Props>();
 
-const codeLines = computed(() => {
-  const code = props.code;
-  const codeLines = code.split("\n");
-  const codeLinesWithEmptyLines = codeLines.map((line) => {
-    if (line === "") {
-      return "\n";
-    }
-    return line;
-  });
-  return codeLinesWithEmptyLines;
-});
+const codeLines = props.code.split("\n");
+for (const line in codeLines) {
+  if (codeLines[line] === "") {
+    codeLines[line] = " ";
+  }
+}
```

Comment:
is this working on your side, @jackoconnordev?


Uploading Screen Recording 2023-07-24 at 12.21.04 PM.mov…